[2024-02-21T07:37:50.241+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-28T00:00:00+00:00 [queued]>
[2024-02-21T07:37:50.265+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-28T00:00:00+00:00 [queued]>
[2024-02-21T07:37:50.265+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:37:50.305+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-01-28 00:00:00+00:00
[2024-02-21T07:37:50.313+0000] {standard_task_runner.py:60} INFO - Started process 79 to run task
[2024-02-21T07:37:50.344+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-01-28T00:00:00+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpy5eizyzq']
[2024-02-21T07:37:50.355+0000] {standard_task_runner.py:88} INFO - Job 69: Subtask get_mail
[2024-02-21T07:37:50.492+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-01-28T00:00:00+00:00 [running]> on host 893865381b0d
[2024-02-21T07:37:50.688+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-01-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-28T00:00:00+00:00'
[2024-02-21T07:37:50.693+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 196, in execute
    context_merge(context, self.op_kwargs, templates_dict=self.templates_dict)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 261, in context_merge
    context.update(*args, **kwargs)
  File "/usr/local/lib/python3.8/_collections_abc.py", line 837, in update
    for key, value in other:
ValueError: too many values to unpack (expected 2)
[2024-02-21T07:37:50.708+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extract_email, task_id=get_mail, execution_date=20240128T000000, start_date=20240221T073750, end_date=20240221T073750
[2024-02-21T07:37:50.739+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 69 for task get_mail (too many values to unpack (expected 2); 79)
[2024-02-21T07:37:50.790+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-21T07:37:50.829+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-21T07:59:31.074+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-28T00:00:00+00:00 [queued]>
[2024-02-21T07:59:31.081+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-28T00:00:00+00:00 [queued]>
[2024-02-21T07:59:31.081+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:59:31.090+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-01-28 00:00:00+00:00
[2024-02-21T07:59:31.094+0000] {standard_task_runner.py:60} INFO - Started process 354 to run task
[2024-02-21T07:59:31.096+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-01-28T00:00:00+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpl1q3ioca']
[2024-02-21T07:59:31.098+0000] {standard_task_runner.py:88} INFO - Job 125: Subtask get_mail
[2024-02-21T07:59:31.125+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-01-28T00:00:00+00:00 [running]> on host 9bab6a0db245
[2024-02-21T07:59:31.181+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-01-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-28T00:00:00+00:00'
[2024-02-21T07:59:34.030+0000] {ImapSmtp.py:839} INFO - IMAP search: 'gmail:'Message from RPA Python' after:15-12-2023'
[2024-02-21T07:59:37.067+0000] {logging_mixin.py:188} INFO - Message from RPA Python0
[2024-02-21T07:59:37.069+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-112.txt')
[2024-02-21T07:59:37.069+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.070+0000] {logging_mixin.py:188} INFO - Message from RPA Python1
[2024-02-21T07:59:37.071+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-113.txt')
[2024-02-21T07:59:37.071+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.072+0000] {logging_mixin.py:188} INFO - Message from RPA Python2
[2024-02-21T07:59:37.073+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-114.txt')
[2024-02-21T07:59:37.073+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.074+0000] {logging_mixin.py:188} INFO - Message from RPA Python3
[2024-02-21T07:59:37.075+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-115.txt')
[2024-02-21T07:59:37.075+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.076+0000] {logging_mixin.py:188} INFO - Message from RPA Python4
[2024-02-21T07:59:37.077+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-116.txt')
[2024-02-21T07:59:37.078+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.078+0000] {logging_mixin.py:188} INFO - Message from RPA Python5
[2024-02-21T07:59:37.079+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-117.txt')
[2024-02-21T07:59:37.080+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.080+0000] {logging_mixin.py:188} INFO - Message from RPA Python6
[2024-02-21T07:59:37.081+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-118.txt')
[2024-02-21T07:59:37.082+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.082+0000] {logging_mixin.py:188} INFO - Message from RPA Python7
[2024-02-21T07:59:37.084+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-119.txt')
[2024-02-21T07:59:37.084+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.084+0000] {logging_mixin.py:188} INFO - Message from RPA Python8
[2024-02-21T07:59:37.086+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-120.txt')
[2024-02-21T07:59:37.086+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.086+0000] {logging_mixin.py:188} INFO - Message from RPA Python9
[2024-02-21T07:59:37.088+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-121.txt')
[2024-02-21T07:59:37.088+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:37.698+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T07:59:37.704+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=extract_email, task_id=get_mail, execution_date=20240128T000000, start_date=20240221T075931, end_date=20240221T075937
[2024-02-21T07:59:37.728+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T07:59:37.739+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
